examples/cli.rs

```diff
@@ -416,6 +433,83 @@ async fn run<C: ConfigStore>(subcommand: Cmd, config_store: C) -> anyhow::Result
             let manager = Manager::load_registered(config_store)?;
             manager.request_contacts_sync().await?;
         }
+        Cmd::ListMessages => {
+            let mut msgs = config_store.messages()?;
+            msgs.sort_by_key(|m| m.metadata.timestamp);
+            for msg in msgs {
+                match msg.body {
+                    ContentBody::DataMessage(message)
+                    | ContentBody::SynchronizeMessage(SyncMessage {
+                        sent:
+                            Some(Sent {
+                                message: Some(message),
+                                ..
+                            }),
+                        ..
+                    }) => {
+                        println!(
+                            "{}: {}",
+                            msg.metadata.sender.e164_or_uuid(),
+                            message.body.unwrap_or_else(|| "".to_string())
```

Comment:
what about filtering only message that have a body in the `match` statement above? this would prevent the output from being littered with empty "system" messages.